packages/preview/psl-thesis/0.1.0/lang.toml

```diff
@@ -0,0 +1,16 @@
+[conf]
+default-lang = "en"
+
+[lang.en]
+conducted_at = "Conducted at"
+presented_by = "Presented by"
+doctoral_school = "Doctoral school n°"
+specialty = "Specialty"
+jury = "Jury members"
+
+[lang.fr]
+conducted_at = "Préparée"
```

Comment:
Not sure, but isn't it supposed to say "Préparée à" ?

Reply:
I've intentionally left out the preposition because it could depend on the institute, e.g. "Préparée au laboratoire XXX", or "Préparée à la Faculté XXX", etc. It's up to the user to add it. Whereas in English it would always be "Conducted at" (but I could be mistaken there). What do you think about it, should I do it differently?

Reply:
Didn't think about that, I think you're right then. Ideally there would be a more powerful translation framework for Typst that allows to change the string depending on the context, but I don't think it exists (yet).